Makefile

```diff
@@ -101,8 +101,9 @@ create-test-db-file: run-migrations
 
 gen-apidoc:
 	rm -fr $(PYDIR)/staticfiles/
+	rm -fr $(APIDOC)
 	apidoc -i $(PYDIR) -o $(APIDOC)
-	cp docs/source/specs/openapi.yml $(APIDOC)/
+	cp docs/source/specs/openapi.json $(APIDOC)/
 
 collect-static:
```

Comment:
First time for me going through this openapi stuff so bear with me.  I had to run `make collect-static` as well to get the `/api/v1/openapi.json` endpoint to return anything.  Should this be part of the `gen-apidoc` Makefile definition?

Reply:
I suppose we could make `gen-apidoc` a target that gets run when you execute `make collect-static`. 🤷‍♂️

Reply:
Maybe.  I'd say whatever would make sure the right thing happens for our OpenShift deployment.  I just wanted to call out that you get a 404 if `make collect-static` isn't ran after generating the api doc.  Wasn't clear what the problem was until I started it up with pdb

Reply:
yeah the openshift does it we are already serving the apidocs and yaml up til now
https://github.com/project-koku/koku/blob/master/openshift/s2i/bin/assemble#L98